src/top-bar/org-unit-select/approval-status-label.module.css

```diff
@@ -0,0 +1,25 @@
+.container {
+    display: grid;
+    align-items: center;
+    grid-template-columns: auto 1fr;
+    column-gap: var(--spacers-dp4);
+}
+
+.iconContainer {
+    display: inline-flex;
+}
+
+.loadingIcon {
+    display: inline-block;
+    width: 16px;
+    height: 16px;
+    border: 2px solid var(--colors-grey500);
+    border-radius: 50%;
+    animation: loading 2s infinite;
```

Comment:
I think we should reduce the animation duration quite a bit. Even on a 2G connection the statuses load fairly quickly. We see the loading state briefly, but we don't really see the animation taking place, making it hard to work out this is a loader. 

Probably starting with a fade in will make this look more like a loader too.